sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1143,3 +1143,12 @@ directive:
   transform: >
     $.put.responses["201"].headers["x-ms-content-crc64"]["x-ms-client-name"] = "ContentCrc64";
 ```
+
+### Rename SetTier to SetAccessTier
+``` yaml
+directive:
+- from: swagger-document
+  where: $["x-ms-paths"]["/{containerName}/{blob}?comp=tier"]
+  transform: >
+    $.put.operationId = "Blob_SetAccessTier";
```

Comment:
This is fine - but most of the time the generated `operationId`s are a best guess and we have to pass a different value.  I wouldn't do a lot of extra work to get around that if it pops up again.